test/transactional-mutations-plugin/transactional-mutations-plugin.test.ts

```diff
@@ -0,0 +1,32 @@
+import { TransactionalMutationsPlugin } from '../../src';
+import { loggerMock } from '../mocks/logger-mock';
+
+let transactionalMutationsPlugin: TransactionalMutationsPlugin;
+
+const setUpContext = (query: string): any => {
+    return {
+        request: {
+            query,
+            operationName: jest.fn(),
+            variables: jest.fn(),
+        },
+        context: jest.fn(),
+    };
+};
+
+describe('transactionalMutationsPlugin tests', () => {
```

Comment:
Check for mutation cases

Reply:
i dont know if i can test this because the whoe logic in there uses the queryRunner object that i am creating inside the function(i can't mock this object)